src/views/assets/Domain/DomainDetail/AssetList.vue

```diff
@@ -60,6 +60,7 @@ export default {
               name: 'AddAsset',
               title: this.$t('Add'),
               type: 'primary',
+              can: !this.$store.getters.currentOrgIsRoot,
               callback: () => {
                 this.$route.params.id = this.object.id
                 this.addAssetSetting.addAssetDialogVisible = true
```

Comment:
There is an additional curly brace at the end of line 60 in line "export default {". Please ensure that it is correct to maintain proper format.